api/app_analytics/influxdb_wrapper.py

```diff
@@ -314,6 +313,46 @@ def get_top_organisations(date_range: str, limit: str = ""):
     return dataset
 
 
+def get_current_api_usage(organisation_id: int, date_range: str) -> int:
+    """
+    Query influx db top used organisations
+
+    :param organisation_id: filtered organisation
+    :param date_range: data range for current api usage window
```

Comment:
I assume this is some kind of proprietary influx date range value, can we either give more information here or (perhaps ideally) accept actual python datetimes and convert to whatever influx needs here?

Reply:
Hmm. Well all the existing functions in this module uses the `date_range: str` pattern and I sorta see why. The module is about being a wrapper around influx db and I don't find that interacting with it is onerous.

Reply:
Given our time again, I'd like to see the influx wrapper require more human understandable inputs, rather than relying on the knowledge of Influx's query language. But... you're right that the rest of the module behaves this way so it's not a hill that I'll die on.

Reply:
This has been created as an issue so we can handle the whole file in one pass, as discussed.

https://github.com/Flagsmith/flagsmith/issues/3417